tests/e2e/support/objects/app-files/link/actions.ts

```diff
@@ -191,6 +192,8 @@ export const changeName = async (args: changeNameArgs): Promise<string> => {
     }
     await sidebar.open({ page: page, resource: resourceName })
     await sidebar.openPanel({ page: page, name: 'sharing' })
+  } else {
+    await openSharingPanel(page)
```

Comment:
why was this required?

Reply:
In `publicLink.feature` when using webUI to create public link for the space, the sharing panel is already opened, which is followed by rename  step in line 135. So on removing the webUI step for creating public link, the test case fails at line 135. So i need to add `await openSharingPanel(page)`
```
134.  And "Alice" creates a public link for the space "team" with password "%public%" using the sidebar panel
135. And "Alice" renames the most recently created public link of space to "spaceLink"
```

Reply:
But as per your review we don't need to use API steps there in `publicLink.feature`, so may be this is not needed.

Reply:
if not used then yes, let's not add it.